### PR TITLE
[4.0] Style install error page

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #18487 .

### Summary of Changes
Styles install error page

image

### CODE REVIEW

**Comment by dgrammatiko** [comment]
@wilsonge hit the green button

File: installation/language/en-US/en-US.ini [hunk 1]
```diff
@@ -201,6 +201,7 @@ INSTL_XML_SUPPORT="XML Support"
 INSTL_ZIP_SUPPORT_AVAILABLE="Native ZIP support"
 INSTL_ZLIB_COMPRESSION_SUPPORT="Zlib Compression Support"
 INSTL_PROCESS_BUSY="Process is in progress. Please wait ..."
+INSTL_ERROR="Error"
```
Comment (brianteeman) [inline installation/language/en-US/en-US.ini:204]: Any reason there is a ! In en-gb but not in en-us

Comment (mbabker) [reply installation/language/en-US/en-US.ini:204]: We don't get as excited over errors as you do?

Comment (brianteeman) [reply installation/language/en-US/en-US.ini:204]: That sounds about right

File: installation/template/error.php [hunk 2]
```diff
@@ -22,56 +22,74 @@
...
-								<?php // Make the first assignment to setError() outside the loop so the loop does not skip Exceptions ?>
-								<?php $this->setError($this->_error->getPrevious()); ?>
-								<?php while ($loop === true) : ?>
-									<p><strong><?php echo JText::_('JERROR_LAYOUT_PREVIOUS_ERROR'); ?></strong></p>
-									<p><?php echo htmlspecialchars($this->_error->getMessage(), ENT_QUOTES, 'UTF-8'); ?></p>
+		<div class="j-install">
+			<!-- Header -->
+			<header class="j-header" role="banner">
+				<div class="j-header-logo">
+					<img src="<?php echo $this->baseurl ?>/template/images/logo.svg" alt="Joomla" class="logo"/>
```
Comment (Quy) [inline installation/template/error.php]: Add ; after `$this->baseurl`

File: installation/language/en-US/en-US.ini (hunk 1 above)
Comment (ciar4n) [reply installation/language/en-US/en-US.ini:204]: My bad ! !  😄

File: installation/template/error.php (hunk 2 above)
Comment (ciar4n) [reply installation/template/error.php]: Ty @Quy

**Comment by wilsonge** [comment]
Thankyou!

**Pull request merged**